packages/system/dracut/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh

```diff
@@ -30,7 +30,7 @@ install() {
     # Include utilities required for cos-setup services,
     # probably a devoted cos-setup dracut module makes sense
     inst_multiple -o \
-        "$systemdutildir"/systemd-fsck partprobe sync udevadm lsblk sgdisk parted mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount
+        "$systemdutildir"/systemd-fsck partprobe sync udevadm lsblk sgdisk parted mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount coreutils
```

Comment:
I think it should just be 
```suggestion
        "$systemdutildir"/systemd-fsck partprobe sync udevadm lsblk sgdisk parted mkfs.ext2 mkfs.ext3 mkfs.ext4 mkfs.vfat mkfs.fat mkfs.xfs blkid e2fsck resize2fs mount xfs_growfs umount basename
```

`coreutils` is the package name, however it ships different binaries, that's indeed what shows in my system:
```
~/_git/kairos examples*
❯ whereis coreutils
coreutils: /usr/lib64/coreutils /usr/share/info/coreutils.info.gz
```
dracut will take care of following symlinks et all, so no need to care much other than specyfing the bin you want

Reply:
ah very cool, didn't know this was possible